tests/http/test_bad_request_with_response_id.py

```diff
@@ -15,15 +15,12 @@ def test_bad_request_with_response_id(self):
         """
         """
 
-        caught_exception = None
-
-        try:
+        with self.assertRaises(ApiException) as caught_exception:
             UserPerson.get(
                 self._get_api_context(),
                 self._INVALID_USER_PERSON_ID
             )
-        except BadRequestException as exception:
-            caught_exception = exception
 
         self.assertIsNotNone(caught_exception)
-        self.assertIsNotNone(caught_exception.response_id)
+        self.assertIsNotNone(caught_exception.exception)
```

Comment:
This assertion seems redundant after using `assertRaises`?

Reply:
true